spec/watir/rspec/matchers/base_matcher_spec.rb

```diff
@@ -168,14 +168,14 @@
         end
       end
 
-      object.should_not be_foo.during(0.1)
-      raised.should be_true
+      expect(object).not_to be_foo.during(0.1)
+      expect(raised).to be_truthy
```

Comment:
Please use be_true here since we care in this test that it is actually true and not anything else, because anything else might mean that functionality is not working correctly.